json-formatter/go/json.go

```diff
@@ -172,7 +115,7 @@ func (self *Formatter) ProcessMessages(reader gio.ReadCloser, stdout io.Writer)
 				}
 				if self.isBackgroundStep(step.Id) {
 					backgroundJsonSteps = append(backgroundJsonSteps, jsonStep)
-					background = self.lookup.LookupBrackgroundByStepId(step.Id)
+					background = self.lookup.LookupBackgroundByStepID(step.Id)
```

Comment:
I see you dropped the `ByID` suffix for some other methods. I think we can drop it here too. If we're not explicit about the key, assume it's by id.

Reply:
that was the idea (assuming default look up is by ID)
This one is a bit different, we lookup by `StepID`, so that's why I kept it there

Reply:
Oh I missed that - sounds good.